BPA/Ch.10/js_super.stan

```diff
@@ -222,24 +241,22 @@ generated quantities {
     int Nind[M];
     int Nalive[M];
 
+    for (i in 1:M)
+      for (t in 1:n_occasions)
+        u[i, t] = z[i, t] * w[i];
```

Comment:
This loop can be replaced with an elementwise product, which is more concise and also more efficient because the expression graph is smaller:

```
u[i] = z[i] .* w[i];
```

if `u` is a matrix, `z` is a matrix, and `w` is a vector.